core/src/main/java/com/nisovin/magicspells/util/data/DataEntity.java

```diff
@@ -4,11 +4,13 @@
 import java.util.HashMap;
 import java.util.function.Function;
 
+import com.nisovin.magicspells.MagicSpells;
 import org.bukkit.entity.Entity;
 import org.bukkit.entity.SpawnCategory;
 import org.bukkit.command.CommandSender;
 
 import com.nisovin.magicspells.util.Util;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
 import org.bukkit.event.entity.EntityDamageEvent;
```

Comment:
Please group and sort imports due to our styling. Seems like before your changes we grouped `EntityDamageEvent` wrongly too.

Reply:
Regrouped the imports